src/ompl/geometric/planners/quotientspace/algorithms/MultiQuotient.h

```diff
@@ -0,0 +1,135 @@
+*     copyright notice, this list of conditions and the following
+*     disclaimer in the documentation and/or other materials provided
+*     with the distribution.
+*   * Neither the name of the University of Stuttgart nor the names
+*     of its contributors may be used to endorse or promote products
+*     derived from this software without specific prior written
+*     permission.
+*
+*  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+*  "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+*  LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+*  FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+*  COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+*  INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+*  BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+*  LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+*  CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+*  LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+*  ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+*  POSSIBILITY OF SUCH DAMAGE.
+*********************************************************************/
+
+/* Author: Andreas Orthey */
+
+#ifndef OMPL_GEOMETRIC_PLANNERS_QUOTIENTSPACE_MULTIQUOTIENT_
+#define OMPL_GEOMETRIC_PLANNERS_QUOTIENTSPACE_MULTIQUOTIENT_
+#include <ompl/geometric/planners/quotientspace/datastructures/QuotientSpace.h>
+#include <type_traits>
+#include <queue>
+
+namespace ompl
+{
+    namespace geometric
+    {
+        /** \brief A sequence of multiple quotient-spaces
+             The class MultiQuotient can be used with any planner which inherits
+             the ompl::geometric::QuotientSpace class.
+
+             Example usage with QRRT
+             (using a sequence si_vec of ompl::base::SpaceInformationPtr)
+             ompl::base::PlannerPtr planner =
+                 std::make_shared<MultiQuotient<ompl::geometric::QRRT> >(si_vec); */
+
+        template <class T>
+        class MultiQuotient : public ompl::base::Planner
+        {
+            typedef ompl::base::Planner BaseT;
+            static_assert(std::is_base_of<QuotientSpace, T>::value, "Template must inherit from Quotient");
+
+        public:
+            const bool DEBUG{false};
+
+            /// \brief Constructor taking a sequence of ompl::base::SpaceInformationPtr
+            ///  and computing the quotient-spaces for each pair in the sequence
+            MultiQuotient(std::vector<ompl::base::SpaceInformationPtr> &siVec, std::string type = "QuotientPlanner");
+            MultiQuotient(ompl::base::SpaceInformationPtr si) = delete;
+            MultiQuotient(ompl::base::SpaceInformationPtr si, std::string type) = delete;
+
+            virtual ~MultiQuotient() override;
+
+            /// Return annotated vertices (with information about QuotientSpace level)
```

Comment:
Can you change the documentation to use `/** \brief <description here> */` style comments?